domenetjenester/uttak/src/main/java/no/nav/foreldrepenger/domene/uttak/uttaksgrunnlag/fp/EndringsdatoRevurderingUtleder.java

```diff
@@ -420,9 +423,10 @@ private EndringsdatoType finnEndringsdatoTypeVedAdopsjon(ForeldrepengerGrunnlag
 
     private Optional<LocalDate> beregnetEndringsdatoBerørtBehandling(UttakInput input, Long utløsendeBehandlingId) {
         var utløsendeUttak = hentUttak(utløsendeBehandlingId).orElseGet(() -> new ForeldrepengerUttak(List.of()));
-        var berørtUttak = hentUttak(input.getBehandlingReferanse().originalBehandlingId());
+        var originalBehandlingId = input.getBehandlingReferanse().originalBehandlingId();
+        var berørtUttak = hentUttak(originalBehandlingId);
         return EndringsdatoBerørtUtleder.utledEndringsdatoForBerørtBehandling(utløsendeUttak,
-            ytelsesFordelingRepository.hentAggregatHvisEksisterer(utløsendeBehandlingId),
+            ytelsesFordelingRepository.hentAggregat(utløsendeBehandlingId),
```

Comment:
Living on the edge? Eller safe

Reply:
burde være safe. Skal ikke være noe som utløser en berørt behandling som mangler yfa